ansible/roles/test/files/ptftests/sonic.py

```diff
@@ -0,0 +1,618 @@
+import ptf
+from ptf.base_tests import BaseTest
+from ptf import config
+import ptf.testutils as testutils
```

Comment:
This is not used. Also overlaps w/ `from ptf.testutils import *`

Reply:
Imports cleaned up.